it/test/controllers/agent/manageBusinesses/add/AddBusinessNameControllerISpec.scala

```diff
@@ -167,7 +167,7 @@ class AddBusinessNameControllerISpec extends ComponentSpecBase {
         IncomeTaxViewChangeStub.stubGetIncomeSourceDetailsResponse(testMtditid)(OK, noPropertyOrBusinessResponse)
 
         When(s"I call GET ${changeBusinessNameShowUrl}")
-        val result = IncomeTaxViewChangeFrontend.getChangeBusinessName(clientDetailsWithConfirmation)
+        val result = IncomeTaxViewChangeFrontend.getChangeBusinessNameNewJourney(clientDetailsWithConfirmation)
```

Comment:
should fix : can we not use `new` in names?. It will make no sense when old ones are removed.